dartagnan/src/main/java/com/dat3m/dartagnan/utils/visualization/ExecutionGraphVisualizer.java

```diff
@@ -34,8 +34,10 @@ public class ExecutionGraphVisualizer {
 	
     private final Graphviz graphviz;
     private Map<Event, String> callStackMapping;
-    private BiPredicate<EventData, EventData> rfFilter = (x, y) -> true;
-    private BiPredicate<EventData, EventData> coFilter = (x, y) -> true;
+    // By default we do not filter anything
+    private BiPredicate<EventData, EventData> rfFilter = (x, y) -> false;
+    private BiPredicate<EventData, EventData> frFilter = (x, y) -> false;
+    private BiPredicate<EventData, EventData> coFilter = (x, y) -> false;
```

Comment:
Any reason you inverted the meaning here? Originally, the meaning was "keep the edge if the filter returns true".
This is in line with, e.g., the meaning of `stream.filter`.

Reply:
I initially found the meaning confusing, but better to follow the "standard" API meaning.